### ISSUE TITLE
Generating configs on Windows has some broken folders due to forward slashes

### ISSUE THREAD

**Issue by Bekey** [myano/jenni#201]
It seems like using ˙˙˙os.getcwd()˙˙˙ on windows causes the config to look like this:

˙˙˙
# Directories to load user modules from
# e.g. /path/to/my/modules
extra = [ 'c:\Users\Bekey\Desktop\buffy/modules/' ]
˙˙˙

which then causes issues because it tries to escape the first character of every folder.

**Comment by kaneda** [comment]
Yeah, all of this should be replaced by the os path separator, simply fix to make this system agnostic
